### DISCUSSION TITLE
How to view the strain as well as edges on a slice?

### DISCUSSION THREAD

**Discussion by Yuanguo1994** [neperfepx/neper#156] [Q&A]
Hello everyone,

 I want to know how to plot the edge, especially the grain edge on a slice? I have done a simulation via fepx. When I need to show the strain distribution as well as the grain boundaries on a slice, it always fail, and only strain is shown on the slice. Could I do add some command to show that? The following is the command I use:

```
neper --rcfile none -V 1_uniaxial_columnar.sim -simstep 4 -datanodecoo coo
-dataelt1drad 0.012 -dataelt3dedgerad 0.0050 -datanodecoofact 10
-dataelt3dcol strain-el33 -slicemesh 'y=0.5' -dataeltscaletitle "Strain 33 (-)"
-dataeltscale 0.000:0.002:0.004:0.006 -showelt1d all
-cameraangle 13.5 -imagesize 400:800 -print top_e33_deform
```

look forward your reply

**Comment by rquey** [comment]
The short answer is that it is not possible right now.

**Reply by saikumar234** [reply]
@rquey , I have the same question. I am trying to see grain boundaries on a slice. I am using the following but I am unable to see the GBs. Is this possible in the latest version in NEPER? Some guidance would be helpful. 

neper -V simulation.tess,dummy.msh  -dataelsetcol real:"file(e.txt)"  -dataelsetcolscheme jet -showcsys 1 -datacsyscoo -0.75:0.65:0.65 -datacsyslabel X:Y:Z -datacsysrad 0.007 -slicemesh z=0.4 -datacellscale 0.0:0.006 -showedge 1 -showscale 1 -showmeshslice 1 -print pers -cameracoo x:y:-5.3 -camerasky 1:0:0 -print zp0p4
